rachel/rachexplore.Rmd

```diff
@@ -47,11 +47,44 @@ world <- getMap(resolution = "low") # Get data for map
     ggtitle("UFO Sightings"))
 ```
 
+```{r}
 
 
+data$sighting_year <- lubridate::year(data$date_time)
 
 
+newdata <- data.frame(data$sighting_year ,data$city_longitude, data$city_latitude )
+newdata2 <- na.omit(newdata) # Get rid of empty rows
 
 
+earlydata <- subset(newdata2, newdata2$data.sighting_year < 2005) # UFO sightings before 2005
+laterdata <- subset(newdata2, newdata2$data.sighting_year > 2005) # UFO sightings after 2005 
```

Comment:
Without an equals sign in one of the filters, we are excluding data from the year 2005